src/WebClient.spec.js

```diff
@@ -35,6 +38,12 @@ describe('WebClient', function () {
       const client = new WebClient();
       assert.instanceOf(client, WebClient);
     });
+
+    it('should build a client with static headers', function () {
```

Comment:
i don't think this is a particularly useful test, since you declared the `staticHeaders` property private. and, if we remove that property as discussed in the comment you added, this would no longer pass. i think we should remove this test case.